src/blockfilter.cpp

```diff
@@ -169,7 +169,7 @@ const std::set<BlockFilterType>& AllBlockFilterTypes()
 
     static std::once_flag flag;
     std::call_once(flag, []() {
-            for (auto entry : g_filter_types) {
+            for (const auto& entry : g_filter_types) {
```

Comment:
Those things keep coming up, so it would be good to have a clang-tidy check for this

Reply:
Added in 4407213ca10a3a28f253fb3e14e28940144f78ce.